index.html

```diff
@@ -0,0 +1,149 @@
+        <hr class="orange-dividing-line-program mx-auto">
+          <div class="desktop-outline-grid-program-container">
+          <div class="grid-program-container">
+            <img src="./images/lecture.png" class=" program-image-icons" alt="illustration showing lecturer presenting">
+            <div class="program-description-title">
+              Lecture
+            </div>
+            <div class="program-description-text">
+              Listen to the speakers from various countries about the messages on community building and innovation
+            </div>
+          </div>
+          <div class="grid-program-container">
+              <img src="./images/exhibition.png" class="program-image-icons" alt="illustration showing an exhibition">
+            <div class="program-description-title">
+              <p>CC Exhibition </p>
+            </div>
+            <div class="program-description-text">
+              Appreciate various creators showing off their locally brewed art pieces.
+            </div>
+          </div>
+          <div class="grid-program-container">
+              <img src="./images/forum.png" class="program-image-icons" alt="illustration showing a discussion">
+            <p class="program-description-title">
+              Forum
+            </p>
+            <p class="program-description-text">
+              Have the time to share your thoughts and opinions with experts for each topic.
+            </p>
+          </div>
+          <div class="grid-program-container">
+              <img src="./images/workshop.png" class="program-image-icons" alt="illustration showing a discussion">
+            <p class="program-description-title">
+              Workshop
+            </p>
+            <p class="program-description-text">
+              Try creating your own creative/ technological work in groups based on your interests.
+            </p>
+          </div>
+          <div class="grid-program-container">
+              <img src="./images/ignite.png" class="program-image-icons" alt="illustration showing a discussion">
+            <p class="program-description-title">
+              Nkwashi Ignite
+            </p>
+            <p class="program-description-text">
+              Get opportunities to network with atendees, affliliates from around the world and the presenters/ authors.
+            </p>
+          </div>
+        </div>
+        <a id="join-conference" class="join-button p-2" href="#" role="button"> JOIN THE NKWASHI CONFERENCE </a>
+      </div>
+    </section>
+
+    <section class="speakers">
+      <h2 class="text-center speaker-title"> Featured Speakers</h2>
+      <hr class="orange-dividing-line-about mx-auto">
+      <div id="speakers" class="grid-speaker-container">
+      </div>
+      <button id="see-less-button" class="see-less-button-hide speakers-button rounded p-2" type="button"> SEE LESS <i class="fas fa-chevron-down see-more-speakers-icon>"></i> </button>
```

Comment:
- [ ] Kindly add a background image to each of the images in the featured-speakers section.

| Orignal design | Yours |
| ------------- | ------ |
| <img src='https://user-images.githubusercontent.com/33323077/125158298-0c0da000-e168-11eb-934c-ae7972113e69.png' width='400px'/> | <img src='https://user-images.githubusercontent.com/33323077/125158311-30697c80-e168-11eb-9ded-97a280e83780.png' width='400px'/> |